firmware/config/stm32f4ems/efifeatures.h

```diff
@@ -246,7 +246,7 @@
 #endif
 
 #ifndef EFI_ENGINE_SNIFFER
-#define EFI_ENGINE_SNIFFER TRUE
+#define EFI_ENGINE_SNIFFER FALSE
```

Comment:
we can't turn this off

Reply:
This stuff can probably be in CCM though - I don't think we hit it with DMA

Reply:
Agree. This is for testing only. STM32F407 is running out of ram. This is temporally solution.